README.md

```diff
@@ -1,2 +1,78 @@
 # rust
-Docker image for Rust
+
+This is a Docker image with Rust 1.18 installed.
+
+## How to Use
+
+In order to use this image for your project, edit the `techio.yml` file and add the following lines to your project:
+
+```yaml
+    runner: techio/rust:latest
+```
+
+## Example
+
+Given the following project:
+
+```
+.
+├── Cargo.toml
+├── techio.yml
+└── src
+    ├── lib.rs
+    ├── string_tests.rs
+    └── uppercase.rs
+```
+
+In this example, the user is asked to write the function `to_upper` (file `uppercase.rs`):
+
+```rust
+pub fn to_upper(string: &str) -> String {
+    return string.to_uppercase();
+}
+```
+
+In order to test the answer, the following unit test is created (file `string_tests.rs`):
+
+```rust
+#[test]
+pub fn test_to_upper() {
+    use uppercase;
+    let expected = "FOO";
+    let actual = uppercase::to_upper("foO");
+    assert_eq!(expected, actual);
+}
+```
+
+and the file `lib.rs`:
+
+```rust
+pub mod string_tests;
+pub mod uppercase;
+```
+
+In the markdown file, the unit test can be called using:
+
+```markdown
+@[Test uppercase]({"stubs":["uppercase.rs"], "command":"cargo test"})
```

Comment:
Don't we have to be more specific in the test we launched? Something like `cargo test string_tests::test_to_upper`?